lib/util.js

```
@@ -0,0 +1,36 @@
+'use strict';
+
+import moment from 'moment';
+
+export default class Util {
+    /**
+     * Flattens uri options given an object of keys and values
+     *
+     * @example `{ foo: 'bar' }` becomes `?foo=bar`
+     *
+     * @param {Object} options - key(s) and value(s)
+     * @returns {string}
+     */
+    static flattenUriOptions(options) {
+        const entries = Object.keys(options);
+        const entriesLength = entries.length;
+
+        if(entriesLength < 1) {
+            return '';
+        }
+
+        return '?' + entries.map((key, index) => options[key] ? `${key}=${options[key]}${index !== entriesLength - 1 ? '&': ''}` : '')
+                            .reduce((previous, item) => previous + item);
+    }
+
+    /**
+     * Converts a date object to a Internet Date/Time Format (see http://tools.ietf.org/html/rfc3339#section-5.6)
+     *
+     * @param {Date} date
+     * @returns {string}
+     */
+    static iso8601(date) {
+        const m = moment(date);
+        return m.isValid() && date !== undefined && date !== null ? m.utc().format('YYYY-MM-DD z') : null;
```

Sounds like it would be best to allow that error to be thrown rather than swallow it. Would let the user know immediately that they've configured their config with an invalid date.